resources-service/src/import/importService.ts

```diff
@@ -0,0 +1,38 @@
+import { ImportApiResource, ImportBatch } from './importTypes';
+import { db } from '../connection-pool';
+import {
+  updateImportProgress,
+  upsertImportedResource,
+  UpsertImportedResourceResult,
+} from './importDataAccess';
+import { AccountSID } from '@tech-matters/twilio-worker-auth';
+
+const importService = () => {
```

Comment:
Can I ask why is it preferred to return the service as a function rather than a module? Dependency injection?

Reply:
Exactly - I know nothing needs to be injected here but I just wanted to stick to a common pattern for the resources service